apsi-enrollment-fe/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:13
+
+WORKDIR /usr/src/app
+
+COPY package.json .
+RUN npm install
+RUN npm install -g @angular/cli
+
+COPY . .
```

Comment:
pytanie dlaczego robimy raz `COPY package.json` instalujemy dependencies, a potem kopiujemy resztę projektu? Znaczy może to być jakiś szczegół, który mnie omija. Nie lepiej od razu skopiować wszystko?

Reply:
Plus ta sama uwaga co do `COPY . .` w backendzie.

Reply:
Kopiujemy package.json, żeby ściągnąć wszystkie zależności, a dopiero potem resztę plików, żeby zbudować.
Można zrobić to razem, ale w ten sposób będziemy ściągać wszystkie zależności za każdym razem (brak cache'a)